x-pack/plugins/enterprise_search/common/constants.ts

```diff
@@ -100,7 +100,7 @@ export const WORKPLACE_SEARCH_PLUGIN = {
 export const SEARCH_EXPERIENCES_PLUGIN = {
   ID: 'searchExperiences',
   NAME: i18n.translate('xpack.enterpriseSearch.searchExperiences.productName', {
-    defaultMessage: 'Enterprise Search',
+    defaultMessage: 'Search experiences',
```

Comment:
My $0.02 I think it would be better to capitalize `Experiences`

Reply:
My gut is telling me it will end up being sentence case `experiences` because it's not a feature, but I asked the UX writing team to weigh in for an official hot take.

Reply:
@TattdCodeMonkey , direct from the UX writers room:

> Hi Julian.  I'd go for consistency here.  Since the other items under the Enterprise Search section are title case, I recommend Search Experiences.  The breadcrumb should also be updated.